### PR TITLE
Update Enable-CsOnlineDialInConferencingUser.md

### PR DESCRIPTION
**Pull request by ruizoscar**
Adding a note to announce the deprecation of the cmdlet and the target date for the deprecation.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

File: skype/skype-ps/skype/Enable-CsOnlineDialInConferencingUser.md [hunk 1]
```diff
@@ -13,6 +13,10 @@ ms.reviewer:
 # Enable-CsOnlineDialInConferencingUser
 
 ## SYNOPSIS
+
+> [!NOTE]
+> This cmdlet will be deprecated on December 31, 2021. To enable Audio Conferencing for a user, please assign an Audio Conferencing license to the user and the user will be automatically enabled. For additional information, see [Assign licenses to users](https://docs.microsoft.com/en-us/microsoft-365/admin/manage/assign-licenses-to-users).
```
Comment (dariomws) [inline skype/skype-ps/skype/Enable-CsOnlineDialInConferencingUser.md:18]:
```suggestion
> This cmdlet will be deprecated on December 31, 2021. To enable Audio Conferencing for a user, please assign an Audio Conferencing license to the user and the user will be automatically enabled. For additional information, see [Assign licenses to users](https://docs.microsoft.com/microsoft-365/admin/manage/assign-licenses-to-users).
```

**Review by dariomws — CHANGES_REQUESTED** [review]
Minor fix.

**Review by get-itips — CHANGES_REQUESTED** [review]
please remove the en-us path from the url, thank you!

**Review by tseward — APPROVED** [review]
Approved pending removal of 'en-us' from the path.

**Comment by yogkumgit** [comment]
@ruizoscar Please implement the suggestions. Thanks

**Comment by yogkumgit** [comment]
@ruizoscar Please update the PR. Thanks

**Comment by yogkumgit** [comment]
@tiburd Kindly take the appropriate next step. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]
Copy review - Approved pending 'en-us' removal from the path.

File: skype/skype-ps/skype/Enable-CsOnlineDialInConferencingUser.md [hunk 2]
```diff
@@ -13,6 +13,10 @@ ms.reviewer:
 # Enable-CsOnlineDialInConferencingUser
 
 ## SYNOPSIS
+
+> [!NOTE]
+> This cmdlet will be deprecated on December 31, 2021. To enable Audio Conferencing for a user, please assign an Audio Conferencing license to the user and the user will be automatically enabled. For additional information, see [Assign licenses to users](https://docs.microsoft.com/microsoft-365/admin/manage/assign-licenses-to-users).
```
Comment (tiburd) [inline skype/skype-ps/skype/Enable-CsOnlineDialInConferencingUser.md:18]:
```suggestion
> This cmdlet will be deprecated on December 31, 2021. To enable Audio Conferencing for a user, please assign an Audio Conferencing license to the user and the user will be automatically enabled. For additional information, see [Assign licenses to users](/microsoft-365/admin/manage/assign-licenses-to-users).
```

**Pull request merged**